x-pack/plugins/observability/public/components/burn_rate_rule_editor/burn_rate_rule_editor.tsx

```diff
@@ -33,106 +35,99 @@ export function BurnRateRuleEditor(props: Props) {
   });
 
   const [selectedSlo, setSelectedSlo] = useState<SLOResponse | undefined>(undefined);
-  const [longWindowDuration, setLongWindowDuration] = useState<Duration>({
-    value: ruleParams?.longWindow?.value ?? 1,
-    unit: (ruleParams?.longWindow?.unit as DurationUnit) ?? 'h',
-  });
-  const [shortWindowDuration, setShortWindowDuration] = useState<Duration>({
-    value: ruleParams?.shortWindow?.value ?? 5,
-    unit: (ruleParams?.shortWindow?.unit as DurationUnit) ?? 'm',
-  });
-  const [burnRate, setBurnRate] = useState<number>(ruleParams?.burnRateThreshold ?? 1);
-  const [maxBurnRate, setMaxBurnRate] = useState<number>(ruleParams?.maxBurnRateThreshold ?? 1);
 
   useEffect(() => {
     const hasInitialSlo = !loadingInitialSlo && initialSlo !== undefined;
     setSelectedSlo(hasInitialSlo ? initialSlo : undefined);
   }, [loadingInitialSlo, initialSlo, setRuleParams]);
 
-  const onLongWindowDurationChange = (duration: Duration) => {
-    setLongWindowDuration(duration);
-    const longWindowDurationInMinutes = toMinutes(duration);
-    const shortWindowDurationValue = Math.floor(longWindowDurationInMinutes / 12);
-    setShortWindowDuration({ value: shortWindowDurationValue, unit: 'm' });
-  };
-
-  const onBurnRateChange = (value: number) => {
-    setBurnRate(value);
-    setRuleParams('burnRateThreshold', value);
-  };
-
   const onSelectedSlo = (slo: SLOResponse | undefined) => {
     setSelectedSlo(slo);
     setRuleParams('sloId', slo?.id);
   };
 
-  useEffect(() => {
-    if (selectedSlo) {
-      const sloDurationInMinutes = toMinutes(toDuration(selectedSlo.timeWindow.duration));
-      const longWindowDurationInMinutes = toMinutes(longWindowDuration);
-      const maxBurnRateThreshold = Math.floor(sloDurationInMinutes / longWindowDurationInMinutes);
-      setMaxBurnRate(maxBurnRateThreshold);
-    }
-  }, [longWindowDuration, selectedSlo]);
+  const isLegacyRule =
+    ruleParams?.windows == null &&
+    ruleParams?.longWindow &&
+    ruleParams?.shortWindow &&
+    ruleParams?.burnRateThreshold;
+
+  const initialWindows = isLegacyRule
```

Comment:
@grabowskit @katrin-freihofner We want to deprecate a few `params` on the alert definition.  Since we are in Beta, do we need to follow the depreciation schedule OR can we just remove them and say "Sorry, your alert doesn't work anymore because it was created during the beta?"